website/build.gradle

```diff
@@ -230,13 +259,13 @@ def getBranchRepo() {
   return "apache/beam/blob/master"
 }
 
-def buildContentDir(name) {
-  "${project.rootDir}/build/website/generated-${name.toLowerCase()}-content"
+def buildContentDir(name, dir = project.rootDir) {
+  "${dir}/build/website/generated-${name.toLowerCase()}-content"
 }
 
 task serveWebsite(type: Exec) {
   dependsOn setupDockerContainer
-  finalizedBy stopAndRemoveDockerContainer
+//  finalizedBy stopAndRemoveDockerContainer
```

Comment:
Why don't we need this anymore? And why do we need it below?

Reply:
Without the changes in `stopAndRemoveDockerContainer`, the container is removed before the files are staged to gcs.

Reply:
Lets remove it rather than commenting it out then